packages/ui-components/src/components/radio/radio.base.scss

```diff
@@ -59,8 +59,9 @@
 	}
 
 	.checked-icon {
-		height: 0;
-		width: 0;
+		height: 8px;
```

Comment:
you checked this change in other behaviors?

Reply:
I tested every usage of this in the component library and it seems to hold up:
https://youtu.be/kv09duiA50M